asset/cpu_triggered_asset_utilization.py

```diff
@@ -88,11 +69,10 @@ def _minutes_to_seconds(minutes):
 
 def loop():
```

Comment:
If you want to follow PEP 8, then 2 blank lines between all functions. https://www.python.org/dev/peps/pep-0008/#blank-lines
"Surround top-level function and class definitions with two blank lines."

This would apply to all of your functions.